openedx/core/djangoapps/schedules/tasks.py

```diff
@@ -42,6 +43,7 @@
 UPGRADE_REMINDER_LOG_PREFIX = 'Upgrade Reminder'
 COURSE_UPDATE_LOG_PREFIX = 'Course Update'
 COURSE_NEXT_SECTION_UPDATE_LOG_PREFIX = 'Course Next Section Update'
+COURSE_DUE_DATE_REMINDER_LOG_PREFIX = 'Course subsection Due Date Reminder'
```

Comment:
```suggestion
COURSE_DUE_DATE_REMINDER_LOG_PREFIX = 'Course Subsection Due Date Reminder'
```

Nit: to match the convention.